src/test/java/competition/subsystems/arm/ArmPositionSolverTest.java

```diff
@@ -0,0 +1,61 @@
+package competition.subsystems.arm;
+
+import competition.BaseCompetitionTest;
+import edu.wpi.first.math.geometry.Rotation2d;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class ArmPositionSolverTest extends BaseCompetitionTest {
```

Comment:
Yay tests! Bonus points for adding one that verifies it crashes with an impossible position so at least we record the known behavior.

Reply:
It gave a result with an invalid input. I've added a special check that the request is at least in range.